src/state/apis/nft/nftApi.ts

```diff
@@ -28,16 +29,40 @@ const moduleLogger = logger.child({ namespace: ['nftApi'] })
 
 type NftState = {
   selectedNftAvatarByWalletId: Record<WalletId, AssetId>
+  nfts: {
+    byId: Record<AssetId, NftItem>
+    ids: AssetId[]
+  }
+  collections: {
+    byId: Record<AssetId, NftCollectionType>
+    ids: AssetId[]
+  }
 }
 const initialState: NftState = {
   selectedNftAvatarByWalletId: {},
+  nfts: {
+    byId: {},
+    ids: [],
+  },
+  collections: {
+    byId: {},
+    ids: [],
+  },
 }
 
 export const nft = createSlice({
```

Comment:
are we going to initialize the nft slice state on app load or at some entrypoint so we have data available and then can fetch data from the slice directly instead of using the return value of the rtk queries like currently?

Reply:
Absolutely, the goal is to fetch items (and hence, base collections) on `<AppContext />` - currently on the last normalization PR, that's still done at component-level, will have to ensure that fetching from `<AppContext />` doesn't overfetch